smithy-build/src/main/java/software/amazon/smithy/build/transforms/TraitRemovalUtils.java

```diff
@@ -16,27 +16,26 @@
 package software.amazon.smithy.build.transforms;
 
 import java.util.HashSet;
-import java.util.List;
 import java.util.Set;
-import software.amazon.smithy.build.ProjectionTransformer;
-import software.amazon.smithy.model.loader.Prelude;
 import software.amazon.smithy.model.shapes.Shape;
 import software.amazon.smithy.model.shapes.ShapeId;
 import software.amazon.smithy.model.traits.Trait;
 import software.amazon.smithy.utils.Pair;
 
-abstract class AbstractTraitRemoval implements ProjectionTransformer {
-    Pair<Set<ShapeId>, Set<String>> parseTraits(List<String> arguments) {
+/**
+ * Utilities for {@link IncludeTraits} and {@link ExcludeTraits}.
+ */
+final class TraitRemovalUtils {
+
+    private TraitRemovalUtils() {}
+
+    static Pair<Set<ShapeId>, Set<String>> parseTraits(Set<String> ids) {
         Set<ShapeId> traitNames = new HashSet<>();
         Set<String> traitNamespaces = new HashSet<>();
 
-        for (String arg : arguments) {
+        for (String arg : ids) {
             if (arg.endsWith("#")) {
                 traitNamespaces.add(arg.substring(0, arg.length() - 1));
-            } else if (arg.equals(Prelude.NAMESPACE)) {
```

Comment:
This seems like a pretty rare case to run in to, but calling out that we're removing this without having deprecated it in 0.9.x.

Reply:
I'll add a PR to master to emit a warning.